patches/golang/all/dns.patch

```diff
@@ -0,0 +1,82 @@
+diff --git a/net/dnsconfig_unix.go b/net/dnsconfig_unix.go
```

Comment:
I would just store this file as `patches/golang_dns.patch`, no need for the long path.